scripts/manage_org.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+#==============================================================================
+#title           :mamage_org.sh
+#description     :This script is a compilation of all the commonly used CCI commands
+#usage           :Under Help
+#=======================================
+
+function help {
+    echo "**This is script written to help QE manage scratch org either for testing or regression purpose **"
+    echo "Below Options can be provided:"
+    echo ' \t '"
+           -o        org to base the new scratch org from
+           -n        new orgname to be created
+           -f        flowname to be applied [ regression/dev/qa]
+           -d        active days
+           -a        action to perform [create/delete/list/info ]
+           -p        additional actions to create an additional usre or to enable rd2 within the org [createuser/rd2] "
+
+    echo "Create a new org or rebuild an org :     ./manage_scratchorg.sh [-o org] [-n orgname] [-f feature] [-d days] [-a create]"
+    echo "Create a new org with rd2 enabled :     ./manage_scratchorg.sh [-o org] [-n orgname] [-f feature] [-d days] [-a create] [-p rd2]"
+    echo "Delete an org:                           ./manage_scratchorg.sh  [-n orgname] [-a delete]"
+    echo "List org based on selection criteria:    ./manage_scratchorg.sh [-w word] [-a list]"
+    echo "Print org info :                         ./manage_scratchorg.sh  [-n orgname] [-a info]"
+    echo "Create Test user :                       ./manage_scratchorg.sh  [-n orgname] [-p createuser]"
+}
+function createusage {
+
+    echo "usage: ./manage_scratchorg.sh [-o org] [-n orgname] [-f feature] [-d days] [-a action]"
+    echo "Example: ./manage_org.sh -o dev -n testrj -f regression -d 5  -a create"
+    echo "Note: Try to provide a unique org name and prefix it with your initials (Eg: rj_regression_org )"
+}
+function deleteusage {
+    clear
+    echo  "\nusage: ./manage_scratchorg.sh  [-n orgname] [-a action]"
+    echo "Example: ./manage_org.sh -n testrj -a delete"
+}
+function infousage {
+
+    echo "usage: ./manage_scratchorg.sh  [-n orgname] [-a action]"
+    echo "Example: ./manage_org.sh -n testrj -a info"
+}
+
+# Prints the list of commandline arguments passed
+function printvariables {
+  clear
+  echo "Creating a scratch org with the below information..."
+  echo "org: $org"
+  echo "orgname: $orgname"
+  echo "flowname: $flowname"
+  echo "daysactive: $daysactive"
+  echo "action: $action"
+  echo "additional params: $params"
+}
+# Deletes the org specified
+function deleteorg {
+  clear
+  count=`cci org list | grep -i -w $orgname | wc -l`
+  if [[ "$count" -ne  1 ]]
```

Comment:
Correct me if I am wrong here. Irrespective of count check, cci commands to delete scratch org will run. Can we put that inside else block?